langserver/config.go

```diff
@@ -27,3 +27,9 @@ type Config struct {
 	// be used.
 	UseBinaryPkgCache bool
 }
+
+func NewDefaultConfig() Config {
+	return Config{
+		MaxParallelism: 8,
+	}
```

Comment:
given we can remove MaxParallelism, rather get rid of new default config then we can just rely on the zero value for Config.

Reply:
Yup, will do.